docs/source/sycamore/transforms/partition.md

```diff
@@ -21,7 +21,7 @@ Parameters:
 
 * ```use_partitioning_service```: If ```True```, the partitioner uses the *Aryn  Partitioning Service*. Defaults to ```True```.
 * ```model_name_or_path```: The HuggingFace coordinates or model local path. It defaults to ```SYCAMORE_DETR_MODEL```, and you should only change it if you are testing a custom model. Ignored when ```use_partitioning_service``` is ```True```.
-* ```threshold```: The threshold to use for accepting the models predicted bounding boxes. A lower value will include more objects, but may have overlaps; a higher value will reduce the number of overlaps, but may miss legitimate objects. It defaults to ```0.4```.
+* ```threshold```: This represents the threshold for accepting the model's predicted bounding boxes. It defaults to "auto", where the model make its best prediction. The user can specify a numerical threshold between 0 and 1. A lower value will include more objects, but may have overlaps, a higher value will reduce the number of overlaps, but may miss legitimate objects. We suggest trying values around ``0.32``.
```

Comment:
Maybe change the recommendation at the end to use auto? You can maybe say something like "We recommend using the default "auto" threshold. If you do set the threshold manually, we recommend starting with a value of 0.32."

Reply:
Updated. thanks!

Reply:
Is the model making the best prediction, or is it using the post processing logic?